spec/features/primary_publishing_org_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.feature "Setting a primary publishing organisation" do
+  scenario "User creates a new document" do
+    given_a_user_belonging_to_an_org_exists
```

Comment:
These steps don’t quite tally with what’s happening in them (create isn’t actually creating). Could we tweak them and make them a bit more succinct?